service/src/main/resources/templates/ehr_link_set_template.mustache

```diff
@@ -0,0 +1,49 @@
+<component typeCode="COMP"{{#isNested}} contextConductionInd="true"{{/isNested}}>
+    <LinkSet classCode="OBS" moodCode="EVN">
+        <id root="{{linkSetId}}" />
+        <effectiveTime>
+            {{#effectiveTimeLow}}<low value="{{effectiveTimeLow}}"/>{{/effectiveTimeLow}}{{^effectiveTimeLow}}<low nullFlavor="UNK"/>{{/effectiveTimeLow}}
+            {{#effectiveTimeHigh}}<high value="{{effectiveTimeHigh}}"/>{{/effectiveTimeHigh}}{{^effectiveTimeHigh}}<high nullFlavor="UNK"/>{{/effectiveTimeHigh}}
+        </effectiveTime>
+        <availabilityTime value="{{availabilityTime}}" />
+        {{#conditionNamed}}
+        <conditionNamed typeCode="NAME" inversionInd="true">
+            <namedStatementRef classCode="OBS" moodCode="EVN">
+                <id root="{{conditionNamed}}"/>
+            </namedStatementRef>
+        </conditionNamed>
+        {{/conditionNamed}}
+        {{#relatedClinicalContent}}
+        <component typeCode="COMP">
+            <statementRef classCode="OBS" moodCode="EVN">
+                <id root="{{.}}"/>
+            </statementRef>
+        </component>
+        {{/relatedClinicalContent}}
+        {{#clinicalStatusIsActive}}<code code="394774009" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="Active Problem">{{/clinicalStatusIsActive}}{{^clinicalStatusIsActive}}<code code="394775005" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="Inactive Problem">{{/clinicalStatusIsActive}}
+            <originalText>{{#clinicalStatusIsActive}}A{{/clinicalStatusIsActive}}{{^clinicalStatusIsActive}}Ina{{/clinicalStatusIsActive}}ctive Problem, {{qualifier}}</originalText>
+            <qualifier inverted="false">
+                {{#qualifierIsMajor}}<name code="386134007" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="Significant"/>{{/qualifierIsMajor}}{{^qualifierIsMajor}}<name code="394847000" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="Unspecified significance"/>{{/qualifierIsMajor}}
+            </qualifier>
+        </code>
```

Comment:
The clinical status code needs to change along with the display. It would be much clearer if the values were assigned in the code and the template had less logic. For example.

```suggestion
<code code="{clinicalStatusCode}" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="{clinicalStatusDisplay}">
  <originalText>{clinicalStatusDisplay}, {problemSignificanceDisplay}</originalText>
  <qualifier inverted="false">
    <name code="{problemSignificanceCode}" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="{problemSignificanceDisplay}"/>
  </qualifier>
</code>
```